stable/goldilocks/values.yaml

```diff
@@ -106,6 +106,8 @@ dashboard:
     hosts:
       - host: chart-example.local
         paths: []
+        # - path: /
+        #   type: ImplementationSpecific
```

Comment:
I don't think these need to be commented. In fact, I'm realizing this will actually be backward compatible if they are not

Reply:
I agree, I think we can uncomment this and remove the `[]` from line 118

Reply:
Fixed